app/src/main/java/com/duckduckgo/app/browser/BrowserViewModel.kt

```diff
@@ -35,13 +35,24 @@ class BrowserViewModel(
         private val queryUrlConverter: OmnibarEntryConverter,
         private val trackerDataProvider: TrackerDataProvider,
         private val trackerDetector: TrackerDetector,
-        private val trackerListService: TrackerListService) :
+        private val trackerListService: TrackerListService,
+        private val duckDuckGoUrlDetector: DuckDuckGoUrlDetector) :
         WebViewClientListener, ViewModel() {
 
+    data class ViewState(
```

Comment:
Make `ViewModel` responsible for deciding what appears in the URL bar; don't let the `Activity` decide

Reply:
I like that you jotted down my random stream of consciousness earlier and turned it into real feedback :)